js/plugins/i18next.js

```diff
@@ -8,8 +8,9 @@ export function install(Vue) {
         return i18n._(value, options);
     });
 
-    Vue.filter('dt', function(value, format) {
-        return value ? moment(value).format(format || 'LLL') : '-';
+    Vue.filter('dt', function(value, format, empty) {
+        empty = empty !== undefined ? empty : '-';
```

Comment:
Why not make the empty string the default value for `empty` and that way not having to pass it each time?